R/pkg/R/DataFrame.R

```diff
@@ -501,6 +501,34 @@ setMethod("createOrReplaceTempView",
               invisible(callJMethod(x@sdf, "createOrReplaceTempView", viewName))
           })
 
+#' Creates a global temporary view using the given name.
```

Comment:
Would you please extend this documentation to explain what is the difference between a global temp view and a normal temp view? I constantly hear from SparkR users that our documentation simply lists the function arguments.